video/out/hwdec/hwdec_cuda.c

```diff
@@ -83,10 +102,6 @@ static int cuda_init(struct ra_hwdec *hw)
     }
     cu = p->cu;
```

Comment:
should `cuda_load_functions` perhaps be moved into there too?

Reply:
I initially thought `cuda_load_functions` was the culprit for the wake up, but it actually wasn't. It's `cuInit` specifically.

Reply:
Sure, but it's extra work that could also be skipped.

Reply:
Good point.

Reply:
Fixed with b80b40fc19f44f6a1ec82747bcf7c852ae7ed0e6